platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/config/VirtualMapConfig.java

```diff
@@ -93,7 +93,7 @@ public record VirtualMapConfig(
         @Min(1) @ConfigProperty(defaultValue = "20") int flushInterval,
         @ConfigProperty(defaultValue = "200000000") long copyFlushThreshold,
         @ConfigProperty(defaultValue = "2000000000") long familyThrottleThreshold,
-        @ConfigProperty(defaultValue = "2") int preferredFlushQueueSize,
```

Comment:
That's the number of cache copies we're talking about here. I don't think that 10000 is a reasonable number for it. It may be 10 but not 10K.

Reply:
this feature meant to be disabled, nevermind then